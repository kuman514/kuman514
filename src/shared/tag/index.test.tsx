import { render } from '@testing-library/react';

import Tag from '.';

describe('Tag', () => {
  it('should show as expected', () => {
    const { container } = render(
      <Tag>If you win you are Maverick, if you lose you are Yasuo.</Tag>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should show as expected 2', () => {
    const { container } = render(
      <Tag customClassName="text-black">Yasuo Maverick</Tag>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
