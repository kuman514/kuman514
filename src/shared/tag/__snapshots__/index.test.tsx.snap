// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag should show as expected 1`] = `
<span
  class="font-mono bg-gray-300 text-red-700 text-sm px-1 py-0.5 rounded-sm undefined"
>
  If you win you are Maverick, if you lose you are Yasuo.
</span>
`;

exports[`Tag should show as expected 2 1`] = `
<span
  class="font-mono bg-gray-300 text-red-700 text-sm px-1 py-0.5 rounded-sm text-black"
>
  Yasuo Maverick
</span>
`;
